css/custom.css

```diff
@@ -188,19 +190,18 @@ span.student-project-name {
   top: -300px;
   padding: 10px;
   width: 269px;
- }
+}
  
- span.student-project-name span {
+span.student-project-name span {
   font-weight: bold;
   text-transform: uppercase;
   font-size: 20px;
   display: table-cell;
   text-align: center;
   vertical-align: middle;
- }
+}
  
- .background-image-holder:hover span.student-project-name {
+.background-image-holder:hover span.student-project-name {
   opacity: 1;
   top: 0;
 }
```

Comment:
add blank line

Reply:
There is a blank line.

Reply:
There is a blank line between every css block.